pyproject.toml

```diff
@@ -21,12 +21,14 @@ classifiers = [
 ]
 dependencies = [
     "aiohttp~=3.11.10",
-    "loguru~=0.7.2",
+    "audioop-lts~=0.2.1; python_version>='3.13'",
+    "loguru~=0.7.3",
     "Markdown~=3.7",
-    "numpy~=1.26.4",
-    "Pillow~=10.4.0",
+    "numpy~=2.1.3",
+    "numba~=0.61.0rc1",
```

Comment:
Btw, this is necessary because `resampy` depends on `numba` but we need a `numba` that's compatible with `numpy` 2.